tensorflow_graphics/projects/pointnet/train_test.py

```diff
@@ -30,13 +30,19 @@ def test_train(self):
       with tempfile.TemporaryDirectory() as logdir:
         sys.argv = [
             "train.py",
-            "--num_epochs", "2",
-            "--assert_gpu", "False",
-            "--ev_every", "1",
-            "--tb_every", "1",
-            "--logdir", logdir,
-            "--batch_size", str(batch_size),
-            ]
+            "--num_epochs",
```

Comment:
Agreed this is ugly :)
Any way to stop the formatter from take action in a range?

Reply:
sorted. There are still a fair few new lines, but at least it's structured logically.